frontend/app/src/components/profitloss/ReportGenerator.vue

```diff
@@ -90,34 +87,22 @@ const accountSettingsRoute = Routes.SETTINGS_ACCOUNTING;
             {{ t('common.actions.generate') }}
           </RuiButton>
         </div>
-        <div class="">
+        <div>
           <RuiMenu
             close-on-content-click
             :popper="{ placement: 'bottom-end' }"
           >
             <template #activator="{ attrs }">
-              <RuiTooltip
-                :open-delay="400"
-                :popper="{ placement: 'top' }"
-                :disabled="isDevelopment && !isDemoMode"
-                class="h-full"
+              <RuiButton
+                class="h-[2.625rem]"
+                size="lg"
+                v-bind="attrs"
               >
-                <template #activator>
-                  <RuiButton
-                    size="lg"
-                    v-bind="attrs"
-                  >
-                    <template #prepend>
-                      <RuiIcon name="lu-bug" />
-                    </template>
-                    <span v-if="isDevelopment && !isDemoMode">
-                      {{ t('profit_loss_reports.debug.title') }}
-                    </span>
-                  </RuiButton>
+                <template #prepend>
+                  <RuiIcon name="lu-bug" />
                 </template>
-
                 {{ t('profit_loss_reports.debug.title') }}
-              </RuiTooltip>
+              </RuiButton>
```

Comment:
The mode with the tooltip was what we did in production and the one with the label is what we do in development, should we just keep the production mode for everything instead?

Reply:
Yeah, I think it doesn't make sense to differentiate.

Reply:
With that I agree but what we use now was previously the develop version right? not the production on? why not keep the production mode instead?